workflow/kfdrc-rnaseq-workflow.cwl

```diff
@@ -175,6 +180,10 @@ inputs:
       \ for Kallisto only if single end input."}
   kallisto_std_dev: {type: 'long?', doc: "Optional input. Standard Deviation of the\
       \ average fragment length for Kallisto only needed if single end input."}
+  annofuse_genome_untar_path: {type: 'string?', doc: "This is what the path will be\
+      \ when genome_tar is unpackaged", default: "GRCh38_v27_CTAT_lib_Feb092018/ctat_genome_lib_build_dir"}
+  annofuse_col_num: {type: 'int?', doc: "column number in file of fusion name."}
```

Comment:
maybe set the default here...unless the wf takes care of it later on

Reply:
taken care of at the tool level.